content/posts/android-betas.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Regarding the Android betas'
+subtitle: 'A small FAQ about our beta programs'
+author: 'Niels van Velzen'
+githubusername: 'nielsvanvelzen'
+date: 2021-07-16T00:00:00-00:00
+justify: 'center'
+---
+<!-- markdownlint-disable MD033 -->
+
+With the release of the first Android TV 0.12 release beta this is a good time to explain how our Android beta programs work and how you can start using it. The mobile app has used a public beta channel for around a year now and with the Android TV app coming close to a new release we're adding a similar program for that as well.
+
+<!--more-->
+
+## What is the difference between release and beta?
+
+A beta version is an early build of an upcoming release. This means that most of the new features are available but there might still be some issues. In the case of the Jellyfin android apps we might release a beta version because we expect issues or want to test something with a larger audience. Generally the beta cycle should only last a short period.
```

Comment:
```suggestion
A beta version is an early build of an upcoming release. This means that most of the new features are available but there might still be some issues. In the case of the Jellyfin android apps, we might release a beta version because we expect issues or want to test something with a larger audience. Generally the beta cycle should only last a short period.
```